### ISSUE TITLE
./play.sh snapshots/space_invaders_200.pkl error out

### ISSUE THREAD

**Issue by mw66** [tambetm/simple_dqn#23]
./play.sh got the rom name wrong, it invokes:

python src/main.py --play_games 1 --display_screen true --load_weights snapshots/space_invaders_200.pkl roms/space.bin

while it should be space_invaders.bin

**Comment by tambetm** [comment]
Yes, this problem was already raised in pull request #6. But I couldn't accept the proposed fix, because it broke the (optional) experiment label feature. I guess different separator should be used for experiment label, for example `-`. I will keep it open until I have time to fix it, you're welcome to submit a pull request.

**Comment by mw66** [comment]
train.sh works :-)

./train.sh roms/space_invaders.bin 

python src/main.py --save_weights_prefix snapshots/space_invaders --csv_file results/space_invaders.csv roms/space_invaders.bin

game=${full%.*}

instead of

game=${full%%_*}
